packages/react-components/react-motion-components-preview/library/src/index.ts

```diff
@@ -1,3 +1,3 @@
-export { Collapse, CollapseSnappy, CollapseExaggerated } from './components/Collapse';
+export { Collapse, CollapseSnappy, CollapseExaggerated, createCollapsePresence } from './components/Collapse';
 export { Fade, FadeSnappy, FadeExaggerated } from './components/Fade';
 export { Scale, ScaleSnappy, ScaleExaggerated } from './components/Scale';
```

Comment:
```suggestion
export type { PresenceMotionFnCreator } from './types';
```

The type should be exported, right? Otherwise it's impossible to create custom definitions.

Reply:
@layershifter We can export `PresenceMotionFnCreator`, that's fine; I was just avoiding exporting anything that wasn't necessary for beta at this point. 

`PresenceMotionFnCreator` isn't particularly special or a gatekeeper; it's a simple pattern of a function that creates a motion function:

```ts
(variantParams?: MotionVariantParams) => PresenceMotionFn<MotionRuntimeParams>;
```